app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,30 @@
 # class for Posts entity
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = current_user
+    @posts = @user.posts
```

Comment:
- [ ] It would be better if we solve the N+1 problem in this case where we want to retrieve a particular user with all post associated to him. This type of query takes so much time to executed hence "N+1" query problem. ***HINT***: We can fix this by using the `includes` method. For example; `@posts = @user.posts.includes(:comments)` this line of code gives us all posts and comments from a specific user and this operation take less time to execute